src/quill-cursors/cursor.ts

```diff
@@ -77,9 +90,26 @@ export default class Cursor {
     this._el.parentNode.removeChild(this._el);
   }
 
+  public toggleNearCursor(e: MouseEvent): boolean {
+    const pointX = e.pageX;
+    const pointY = e.pageY;
+
+    const gap = isTouchDevice() ? 16 : 8;
+
+    const {x, y, height} = this.coordinates;
+
+    const isXNear = pointX - x < gap && pointX - x > -gap;
+    const isYNear = pointY - y - height < (gap / 2) && pointY - y > -(gap / 2);
+    const shouldShow = isXNear && isYNear;
```

Comment:
We can get rid of all this maths if we just put the padding back on the element.

Reply:
No, we can't, because it's used for touch events. But I simplified the math